demo-shell/src/app/components/cloud/process-details-cloud-demo.component.html

```diff
@@ -11,6 +11,8 @@ <h4 data-automation-id="process-details-header">Simple page to show the process
         class="app-cloud-layout-overflow"
         [appName]="appName"
         [processInstanceId]="processInstanceId"
+        [showContextMenu]="true"
+        [showMainDatatableActions]="true"
```

Comment:
Are we sure this is needed? It is not in the scope of the epic to change the demo shell